docs/content/documentation/deployment/sourcehut.md

```diff
@@ -0,0 +1,53 @@
++++
+title = "Sourcehut Pages"
+weight = 15
++++
+
+Deploying your static Zola website on [Sourcehut Pages][srht] is very simple.
+
+You only need to create a manifest `.build.yml` file in your root folder of your Zola project and push your changes to the Sourcehut git/hg repository. To create your `.build.yml` you can start by [a template][srht-tpl].
+
+Example:
+
+``` yaml
+image: alpine/edge
+packages: [ zola ]
+oauth: pages.sr.ht/PAGES:RW
+environment:
+  site: www.example.org
+sources:
+  - https://git.sr.ht/~your_username/my-website
+tasks:
+  - build: |
+      cd my-website
+      zola build
+  - package: |
+      cd my-website
+      tar -C public -cvz . > ../site.tar.gz
+  - upload: |
+      acurl -f https://pages.sr.ht/publish/$site -Fcontent=@site.tar.gz
+```
```

Comment:
Can you pin the zola version somehow?

Reply:
Let's ask @fredrik-lindseth who provided [the patch](https://github.com/getzola/zola/pull/1251) where I learned that Zola is available as Alpine package :-)

Reply:
@Keats ok, after some searches I can confirm I can't pin a specific package version.

Each [Alpine release](https://pkgs.alpinelinux.org/packages?name=zola&branch=edge) tipically has one version of a package which receives minor updated for bug or security fixes.

`alpine/edge` instead also receives also release updates for a package.

(I think your question was about using a "unstable" Zola release in a CI)